db/migrate/20170413221549_counter_caches.rb

```diff
@@ -4,7 +4,7 @@ def change
     add_column :members, :harvests_count, :integer
     add_column :members, :seeds_count, :integer
 
-    Member.find_each do |member|
+    Member.unscoped.find_each do |member|
```

Comment:
I had to modify this old migration. The models now want the deleted_at column to exist, and it doesn't yet at the point in the migration.